.github/workflows/test.yml

```diff
@@ -145,3 +145,29 @@ jobs:
       features: "${{ toJSON(matrix.features) }}"
       container_env: "${{ toJSON(matrix.env) }}"
       repo: "${{ vars.DOCKERHUB_REPOSITORY || github.repository }}"
+
+  build-rapids:
```

Comment:
Do we want to move this to a separate workflow that is triggered nightly before merging?

Reply:
I like having it in test.yaml. We can make it where it's only run if things that affect the RAPIDS projects (like `devcontainer-utils` or `rapids-build-utils` features) are changed.

Reply:
I thought the purpose of this PR would be to have a nightly all-RAPIDS build. Is there a way to add a trigger for that?

Reply:
Sure, we can make it a reusable workflow.